.github/actions/config-variations/action.yml

```diff
@@ -0,0 +1,119 @@
+# SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+
+name: 'Cuatom config tests'
+description: 'Build and test mldsa-native with various custom configs'
+inputs:
+  gh_token:
+    description: 'GitHub token'
+    required: true
+  tests:
+    description: 'List of tests to run (space-separated IDs) or "all" for all tests. Available IDs: pct-enabled, pct-enabled-broken, custom-zeroize, no-asm, custom-randombytes, custom-memcpy, custom-memset, custom-stdlib'
+    required: false
+    default: 'all'
+  opt:
+    description: 'Optimization level to pass to multi-functest'
+    required: false
+    default: 'all'
+runs:
+  using: 'composite'
+  steps:
+    - name: "PCT enabled"
+      if: ${{ inputs.tests == 'all' || contains(inputs.tests, 'pct-enabled') }}
+      uses: ./.github/actions/multi-functest
+      with:
+        gh_token: ${{ inputs.gh_token }}
+        compile_mode: native
+        cflags: "-DMLD_CONFIG_KEYGEN_PCT -fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all"
+        func: true
+        kat: true
+        acvp: true
+        opt: ${{ inputs.opt }}
+        examples: true
+        extra_args: ""
```

Comment:
I see a warning in CI saying 
`Warning: Unexpected input(s) 'extra_args', valid inputs are ['nix-shell', 'custom_shell', 'nix-cache', 'nix-verbose', 'gh_token', 'cflags', 'compile_mode', 'opt', 'func', 'kat', 'acvp', 'examples', 'stack']`

This should be removed.